src/app/api/me/route.ts

```
@@ -0,0 +1,12 @@
+import { NextResponse } from 'next/server'
+import { SessionPayload, withAuth } from '@/app/api/session'
+
+async function getUser(session: SessionPayload) {
```

ESLint

[error] 'session' is defined but never used.

(@typescript-eslint/no-unused-vars)